_installation/interoperability.md

```diff
@@ -0,0 +1,112 @@
+
+
+### Babelfish/PostgreSQL Interoperability 
+
+Babelfish for PostgreSQL provides two endpoints through which the Babelfish database can be accessed: 
+
+- the TDS port (by default, 1433) providing a T-SQL interface with T-SQL syntax and semantics. TDS and T-SQL are used as synonyms for SQL Server SQL dialect as well as the corresponding connection to the database.
+
+- the PostgreSQL port (by default, 5432) providing a PL/pgSQL interface with PL/pgSQL syntax and semantics. PostgreSQL and PL/pgSQL are used as synonyms for the PostgreSQL SQL dialect as well as the corresponding connection to the database.
+
+The Babelfish database is a PostgreSQL database created for the purpose of emulating the SQL Server multi-DB environment in the T-SQL dialect, with a `master` and `tempdb` database and one or more user databases. The default name of this database is `babelfish_db`. A TDS connection is placed inside this Babelfish database that cannot be accessed through T-SQL. There can only be a single Babelfish database per PostgreSQL instance.
+
+Since the Babelfish database is ultimately a PostgreSQL database, it is technically possible to access it from the PostgreSQL port and run PL/pgSQL statements against it. However, you should exercise caution when accessing the Babelfish database through the PostgreSQL port. 
+
+From a Babelfish perspective, it is safest to avoid accessing the Babelfish database from PostgreSQL altogether. However, there may be valid reasons for wanting to use PostgreSQL functionality. For example, if a particular aspect of T-SQL is not supported by Babelfish, you may be able to implement similar behavior natively in PostgreSQL to allow application migration to PostgreSQL. This comes with certain implications and risks, as described below.
+
+Considerations apply whenever the execution context is not pure TDS/T-SQL; for example, when PostgreSQL SQL statements are executed in a Babelfish context. This is the case when:
+
+- In a PostgreSQL connection, PostgreSQL statements are executed against objects created on the TDS port (like tables, views, and procedures). 
+
+- In a TDS connection, T-SQL statements are executed against objects created on the PostgreSQL port (like tables, views, and procedures), or when PostgreSQL built-in functions are called from T-SQL. Based on customer feedback, this seems to be a common scenario.
+
+In both cases, the SQL dialect is temporarily (and transparently) switched in the session while accessing/executing objects created in a different SQL dialect.
```

Comment:
the original text had the word "effectively" prior to switched. This matters because we should avoid the suggestion that the dialect truly switches  which may suggest that this is something that can be done by users. We must steer clear of even the slightest suggestion in that direction. The current text over-simplifies it